Blind75/Blind75-1-Two-Sum/src/main/Main.java

```diff
@@ -0,0 +1,13 @@
+public class Main {
+    public int[] twoSum(int[] nums, int target) {
```

Comment:
Explain time and space complexity and why it is or isn't optimal.

Reply:
Added a new markdown file in the file structure explaining efficiency and how I would make it more efficient.